Readme.md

```diff
@@ -227,9 +225,211 @@ See package **Neos.EventStore**.
+
+/**
+ * General purpose Organization Read Model
+ *
+ * @Flow\Entity
+ * @ORM\Table(name="acme_projection_organization_v2")
+ */
+class Organization
+{
+
+    /**
+     * @Flow\Inject
+     * @var AssetRepository
+     */
+    protected $assetRepository;
+
+    /**
+     * @ORM\Id
+     * @var string
+     */
+    public $identifier;
+
+    /**
+     * @var string
+     */
+    public $name;
+
+    /**
+     * @ORM\Column(nullable=true)
+     * @var string
+     */
+    public $logoIdentifier;
+
+    /**
+     * @var array
+     */
+    public $projects = [];
+
+    /**
+     * @return AssetInterface|null
+     */
+    public function getLogo()
+    {
+        if ($this->logoIdentifier !== null) {
+            return $this->assetRepository->findByIdentifier($this->logoIdentifier);
+        }
+        return null;
+    }
+}
+```
+
+Read Models currently need to be annotated with `@Flow\Entity`. At a later point it is planned to introduce a specific
+annotation `@CQRS\ReadModel` for that, but that requires a core change in Flow.
+
+It is best practice to manually set the database table name via the `@ORM\Table` annotation.
+
+Like with Entities, injected properties or those marked with `@Flow\Transient` will be ignored by the persistence
+mechanism.
+
+You can define one or more properties which are used as the identifier (or a compound identifier) by using the
+`@ORM\Id` annotation.
+
+Property types are detected by Flow's Class Schema implementation. You can override this base configuration through
+`@ORM\Column` annotations.
+
+In general, properties are *public* for easier handling in code dealing with Read Models. Even if a user decides to
+modify a property, it won't be persisted, because the `update()` method in the Projector can only be called by the
+Projector itself. This Read Model provides a special getter for retrieving the `Asset` object of an organization's
+logo.
+
+A common use case for Read Models are Fluid templates: simply access any of the properties (including `logo`) by
+passing the Read Model instance as a variable.
+
+The database schema for these models / projections needs to be created with Flow's regular Doctrine migration mechanism.
+That means: `./flow doctrine:migrationgenerate`, adjust and `./flow doctrine:migrate`.
+
+The corresponding Projector class for this example projection could look like this:
+
+```php
+namespace Acme\Application\Projection\Organization;
+
+use Acme\Domain\Aggregate\Organization\Event\OrganizationHasBeenCreated;
+use Acme\Domain\Aggregate\Organization\Event\OrganizationHasBeenDeleted;
+use Acme\Domain\Aggregate\Organization\Event\OrganizationLogoHasBeenChanged;
+use Neos\Cqrs\Projection\Doctrine\AbstractDoctrineProjector;
+use TYPO3\Flow\Annotations as Flow;
+
+/**
+ * Organization Projector
+ *
+ * @Flow\Scope("singleton")
+ */
+class OrganizationProjector extends AbstractDoctrineProjector
+{
+    /**
+     * @Flow\Inject
+     * @var OrganizationFinder
+     */
+    protected $finder;
```

Comment:
good idea, but isn't it a bit confusing to have
```
Organization.php
Organizations.php
```
on the same level?
And what if you have some readmodel that just represents a single instance like `OrganizationOfTheMonth`?